application/server/configuration_test.go

```diff
@@ -323,7 +322,7 @@ func Test_UpdateSettings(t *testing.T) {
 		assert.Equal(t, config.DefaultDeeproxyApiUrl, c.SnykCodeApi())
 	})
 
-	t.Run("blank organisation is ignored", func(t *testing.T) {
+	t.Run("blank organization is ignored", func(t *testing.T) {
```

Comment:
Well, this is gonna be contentious: US vs. British spelling. If we do these kind of changes, we should align on the dictionary to be used...

Reply:
We are... it's the one used by the `misspell` lint rule. To be fair, I spell "cancellation" with two L's and its weird for me as well.

Overall I think getting a consistent spelling out of this is better than any particular style -- or code reviewers having to nitpick spelling.

However there's a style guide for user documentation which also serves as precedent here ([internal link](https://snyksec.atlassian.net/wiki/spaces/WS/pages/1837564002/Snyk+Style+Guide)).